backend/tests/conftest.py

```
@@ -0,0 +1,605 @@
+# Mock FastAPI testclient
+class MockResponse:
+    def __init__(self, status_code=200, json_data=None, text="", headers=None):
+        self.status_code = status_code
+        self._json_data = json_data or {}
+        self.text = text
+        self.headers = headers or {}
+
+    def json(self):
+        return self._json_data
+
+class MockTestClient:
```

Ruff

Redefinition of unused `MockTestClient` from line 83

(F811)